requirements/env_hdk.yml

```diff
@@ -2,19 +2,18 @@ name: modin_on_hdk
 channels:
   - conda-forge
 dependencies:
+  - pip
+
+  # required dependencies
   - pandas>=2,<2.1
-  - pyarrow
   - numpy>=1.18.5
-  - fsspec
-  - pip
-  - pytest>=6.0.1
-  - pytest-cov>=2.10.1
-  - pytest-xdist>=2.1.0
-  - coverage
-  - pygithub
   - pyhdk==0.7
-  - s3fs>=2021.8
+  - fsspec
+  - packaging
   - psutil
+
+  # optional dependencies
+  - s3fs>=2021.8
```

Comment:
Could you please also try to remove these guys
  - scipy
  - xgboost>=1.7.1,<2.0.0
  - scikit-learn-intelex
  - matplotlib
 
I don't think they are really on proper place and should be removed

Reply:
I was only able to remove xgboost and scikit-learn-intelex.

scipy and matplotlib are using in unit tests.

Reply:
@Garra1980 ready for review

Reply:
Thanks!